### ISSUE TITLE
Generic heating/warm keeping icon

### ISSUE THREAD

**Issue by frenck** [Templarian/MaterialDesign#6325]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

There are a couple of icons related to heating, for example

![image](https://user-images.githubusercontent.com/195327/141646128-e1ceb91c-7b3f-4ddf-8724-c55e1ef171b5.png)

However, a generic heat or keep warm icon doesn't exist. This could be useful for a lot of things that don't have a specific icon. For example, keep boiler warm, keep water heater warm, keep kettle heated or warm, or maybe even some coffee warm holder cup thingy.

### Give us an example of what this icon might look like.

Basically, it would look like the top part of radiator heating:

![image](https://user-images.githubusercontent.com/195327/141646187-38b46f66-0f43-4b66-b606-9936e6f94f81.png)

![image](https://user-images.githubusercontent.com/195327/141646208-e5241323-3ea8-41b8-a49b-c312f69692cb.png)

**Comment by goyney** [comment]
I'm not sure what to call this other than "heat". Thoughts?

![image](https://user-images.githubusercontent.com/1015040/141667303-8325b78f-7e0e-4a84-830b-d193d74b52b8.png)
[heat.svg.zip](https://github.com/Templarian/MaterialDesign/files/7533174/heat.svg.zip)

**Comment by frenck** [comment]
"Heat" sounds fitting.

Warm or warmth could work too.

**Comment by frenck** [comment]
Heat-wave maybe?

**Comment by DannyS86** [comment]
I've simply searched for "heat" in expectation to find this icon. I think "heat" would be enough. "heat-wave" sounds good too - "wave" was another keyworded I searched for. Can't wait for this icon, I need it in contrast to cold, for what I used the simple snowflake. Currently there is no icon I would use instead.

**Comment by goyney** [comment]
Added.

**Comment by frenck** [comment]
Thank you @goyney ❤️
